geoapps_utils/driver/data.py

```diff
@@ -39,28 +39,28 @@ class BaseData(BaseModel):
     title: str
     workspace_geoh5: Optional[Workspace] = None
 
-    @classmethod
-    def _parse_input(cls, input_data: dict[str, Any]) -> dict[str, Union[dict, Any]]:
+    @staticmethod
+    def model_to_dict(
```

Comment:
Maybe name it something more descriptive? `collect_input_from_dict`?

Reply:
done